cmd/placement-translator/main.go

```diff
@@ -104,6 +104,9 @@ func main() {
 	fs.IntVar(&concurrency, "concurrency", concurrency, "number of syncs to run in parallel")
 	espwClientOpts := NewClientOpts("espw", "access to the edge service provider workspace")
 	espwClientOpts.AddFlags(fs)
+	rootClientOpts := NewClientOpts("root", "access to root workspace")
+	rootClientOpts.overrides.CurrentContext = "root"
+	rootClientOpts.AddFlags(fs)
```

Comment:
Please keep the documentation consistent with the code.  Do not forget about https://docs.kubestellar.io/main/Coding%20Milestones/PoC2023q1/placement-translator/#usage

Reply:
Done